mitiq/shadows/shadows_utils.py

```diff
@@ -88,16 +146,28 @@ def n_measurements_opts_expectation_bound(
 
 
 def fidelity(
-    state_vector: NDArray[np.complex64],
+    sigma: NDArray[np.complex64],
     rho: NDArray[np.complex64],
 ) -> float:
     """
-    Calculate the fidelity between a state vector and a density matrix.
+    fidelity is a measure of the "closeness" of two quantum states.
+    It expresses the probability that one state will pass a test to
+    identify as the other.
     Args:
-        state_vector: The vector whose norm we want to calculate.
-        rho: The operator whose norm we want to calculate.
+        sigma: Quantum state.
+        rho: Quantum state
```

Comment:
to check

Reply:
Oh, Andrea, here $\sigma$ and $\rho$  might not be a quantum state... I left some comments for why it is on #1942 under your comment, please check whether you are happy with my comments.  I have change it to
```
sigma: A state in terms of square matrix or vector.
rho: A state in terms square matrix or vector.
```

Reply:
@Min-Li please link directly to your comment regarding this instead of the PR.

Reply:
Directly linking the comment is always better. In any case, the current docstring for sigma end rho seems good to me.